src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/entity/PlayerBeanValidationTest.java

```diff
@@ -21,32 +28,59 @@ void setup() {
     validatorFactory.close();
   }
 
-  @Test
-  void valid_player_is_accepted() {
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("playerNames")
+  void valid_player_is_accepted(String name, ValidationListMatcher validationListMatcher) {
     Player player = new Player();
-    player.setName("test");
+    if (name != null) {
```

Comment:
Same questions here as for the GameBeanValidationTest.

Reply:
ecause if the name is null, player.setName throws IllegalArgumentException.
And the name of the game is anyway null if it was never set.